xbmc/addons/kodi-addon-dev-kit/include/kodi/xbmc_pvr_types.h

```diff
@@ -80,6 +80,7 @@ struct DemuxPacket;
 #define PVR_STREAM_PROPERTY_STREAMURL "streamurl" /*!< @brief the URL of the stream that should be played. */
 #define PVR_STREAM_PROPERTY_INPUTSTREAMADDON  "inputstreamaddon" /*!< @brief the name of the inputstream add-on that should be used by Kodi to play the stream denoted by PVR_STREAM_PROPERTY_STREAMURL. Leave blank to use Kodi's built-in playing capabilities. */
 #define PVR_STREAM_PROPERTY_MIMETYPE "mimetype" /*!< @brief the Mime-Type of the stream that should be played. */
+#define PVR_STREAM_PROPERTY_START_OFFSET "startoffset" /*!< @brief the start offset of the stream that should be played. */
```

Comment:
Please add the scale of the offset to the documentation. Is it microseconds, seconds, ... ?

Reply:
Done